---
geometry_msgs/PoseStamped corridor_pose
bool success
